### PR TITLE
More documentation tweaks

### PR DESCRIPTION
**Pull request by ghaskins**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,35 +191,69 @@ Repeat the --init-token flow to set up a fresh HSM instance.
 
 ## login
 
-The login subcommand allows you to create an access token useful for invoking Manetu APIs under the identity of a Service.   Thus, the use of the command has a prerequisite on an existing Service Account registered with the matching public key of the security token you intend to use.
+The login subcommand allows you to create an access token for invoking Manetu APIs under the identity of a Service via the OAUTH [private_key_jwt](https://openid.net/specs/openid-connect-core-1_0-15.html#ClientAuthentication) authentication flow.   Thus, the use of the command has a prerequisite on an existing Service Account registered with the matching public key of the security token you intend to use.
 
 Two types of security tokens are supported: _hsm_ and _pem_.  Each class has options common to both and unique to the chosen type.
```
Comment (muralisrini) [inline README.md:196]: Now that we are cleaning this up... to me "hsm" and "pem" are not really tokens are they? Not sure of the best way to characterize them .. are they "modes" or "mechanisms" ?

Comment (ghaskins) [reply README.md:196]: Well, at least historically, the basic notion was that an HSM with a Manetu-compatible credential was the "security token" that you would install in a server.  The usage is reflected here, where the " token " type can be either HSM or PEM based.  I do not know if we have applied that definition consistently or should continue to do so.

Comment (muralisrini) [reply README.md:196]: The response from the server to a login is a (also) a token. And the server itself just requires a certificate to define the ID and does not differentiate and does not treat he cert as a "token" - which is what `hsm` and `pem` help to define.

I could be missing the core of what we refer to as "security token". Not a biggie, just helps to clarify notions for me.

Comment (ghaskins) [reply README.md:196]: FWIW, the intro paragraph tries to define it:

```
It supports creating, reading, enumerating, and deleting "security tokens," simply a public/private key pair and a self-signed x509.
```

Comment (ghaskins) [reply README.md:196]: In retrospect, calling them something like "service-account credentials" would have been better, perhaps.

Comment (muralisrini) [reply README.md:196]: Its a nit, just something that's been niggling .. as u point out, it won't cause any confusion on the part of users. Resolving.

**Review by muralisrini — APPROVED** [review]
lgtm (ty for the discussion).

**Pull request merged**